bivrost-solidity-types/src/main/kotlin/pm/gnosis/model/Solidity.kt

```diff
@@ -825,18 +828,12 @@ object Solidity {
             return SolidityBase.DynamicType.Parts(length, contents)
         }
 
-        override fun equals(other: Any?): Boolean {
```

Comment:
We should probably generate these (as proper comparision will not work otherwise)

Reply:
Agreed. Can be part of a different issue/PR 👍